python/rpdk/python/__init__.py

```diff
@@ -1,5 +1,5 @@
 import logging
 
-__version__ = "2.1.1"
+__version__ = "2.1.2"
```

Comment:
releasing both?

Reply:
why not. need to update the template for requirements anyway to be less strict